client/src/pages/AssignmentBoard.jsx

```diff
@@ -78,6 +78,12 @@ export default function AssignmentBoard() {
     const [volunteerCards, setVolunteerCards] = useState([]);
     const [projectCards, setProjectCards] = useState([]);
 
+    if (!JSON.parse(localStorage.getItem('tasksComplete'))) {
+        localStorage.setItem("tasksComplete", JSON.stringify({viewed: [], notViewed: []}))
+    }
+    let {viewed} = JSON.parse(localStorage.getItem('tasksComplete'));
```

Comment:
To make this more React-y, you could store these as state variables. You could initialize them as an empty array, and then use a `useEffect` to parse the values from localStorage when the page loads.